### ISSUE TITLE
Since 3.0.7x update WMC no longer communicating with Emby Server

### ISSUE THREAD

**Issue by mikey46656** [MediaBrowser/Emby#2201]
As stated in the heading, v3.0.71 and 3.0.72 no longer communicate properly with WMC. When trying to run MB from the WMC menu as usual (and as has been working flawlessly for several YEARS) I now continuously get the error message "emby server is not running", despite the fact that it clearly is running because I can look at it, check the settings, browse the library from Chrome, etc., but WMC still insists that the server is not running.

This would not be a problem if it were possible to roll back to a v3.0.6x release, so please, please, please offer rollbacks as an option; I know people have been asking for this for several years now. and it would be greatly appreciated.

Thanks.

**Comment by LukePulverenti** [comment]
Hi, are you still having an issue? I believe you created the same topic in the Emby community, can you link me to that? Ebr suggested it is probably due to your server potentially changing ip address. Thanks.

**Comment by mikey46656** [comment]
Hi. Thanks very much for taking a look at this. I did also put it in the community forum, but don't have the address handy (was done from my htpc). The IP address change sounds feasible, but the problem had been happening erratically for more than a month before it became "permanent", meaning not solvable by restarting anymore.

Anyway, I've since found and installed Emby for WMC, which seems to be working perfectly, though it appears to be older(?). We use our htpc ~90% of the time for watching and recording TV, so I do want to stick with WMC/Win7. If you think there's a compelling reason to go back to the other Emby server then I'm willing to have a go at reverting and then trying to fix the (possible) IP problem, but otherwise, I'm willing to close the issue as long as Emby for WMC keeps working. A couple of times a week we use it to watch something from our collection of (legally) ripped DVDs, for which we love its library management and synopsis display, but new features are not really crucial to us.

Thanks again!
-m
